medusa/providers/torrent/json/t411.py

```diff
@@ -375,10 +375,10 @@ def login(self):
             return False
 
         if response and 'token' in response:
-            self.token = response['token']
+            self.token = str(response['token'])
             self.tokenLastUpdate = time.time()
             # self.uid = response['uid'].encode('ascii', 'ignore')
-            self.session.auth = T411Auth(self.token)
+            self.headers.update({'Authorization': self.token})
```

Comment:
This should be `self.session.headers.update(...)`.  Ofc this makes the assumption that headers are properly used in the superclass.